src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -4,7 +4,11 @@
 import com.meilisearch.sdk.model.Task;
 import java.util.HashMap;
 
-/** Wrapper around the HttpClient class to ease usage for Meilisearch indexes */
+/**
+ * Wrapper around HttpClient class to use for Meilisearch indexes
```

Comment:
Is it necessary to say that it wraps httpClient? Woulnd't it not be better to just state that it is the `Indexes API` ?

```suggestion
 * Meilisearch Indexes class
```

```suggestion
 * Indexes class covering all the methods related to an index
```